apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -128,16 +136,13 @@ const TutorialDetail = React.createClass({
                 className="modal-body"
                 style={styles.tutorialDetailModalBody}
               >
-                <div className="col-50">
+                <div style={styles.tutorialDetailImageContainer} className="col-xs-12 col-sm-6">
```

Comment:
I see we're still using bootstrap for some of our responsiveness. How do we decide when to use/not use?